pom.xml

```diff
@@ -336,6 +337,17 @@
                 <artifactId>metrics-reporter</artifactId>
                 <version>${version.microservice.metrics-reporter}</version>
             </dependency>
+            <dependency>
+                <groupId>gov.nsa.datawave.microservice</groupId>
+                <artifactId>query-metric-api</artifactId>
+                <version>${version.microservice.query-metric-api}</version>
+                <exclusions>
```

Comment:
I'm curious why the exclusions are necessary as it's a bit different from the other microservice dependencies.  Maybe it's worth commenting.

Reply:
Have to exclude transitive webservice dependencies and two others to avoid version conflicts.  I pushed changes to limit the exclusions.